virttest/tests/unattended_install.py

```diff
@@ -433,6 +433,12 @@ def answer_kickstart(self, answer_path):
             content = "\n".join(lines)
             contents = re.sub(dummy_repos_re, content, contents)
 
+        dummy_pkgs_re = r'\bKVM_TEST_PKGS\b'
+        if re.search(dummy_pkgs_re, contents):
+            # Extra packages to be installed and locked"
```

Comment:
What do you think about add a comment that,  Use `space` as a separator for multiple pkgs

Reply:
I think it is a good idea, thanks :slightly_smiling_face:  (Added.)